packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -150,6 +151,24 @@ String getGradleVersionForAndroidPlugin(Directory directory) {
   return getGradleVersionFor(androidPluginVersion);
 }
 
+const int _kExecPermissionMask = 0x8049; // u+x
```

Comment:
Is this a+x ? It might also be good to translate this to octal in the comment `0o100111`. What is the leftmost bit for, by the way?

Reply:
That's right. sorry. Updated the comment to `a+x`.  We don't need the leftmost bit. I realized  `File.statSync().mode` includes file permissions in the lower 16 bits.